docs/content/guides/getting-started/react-methods.md

```diff
@@ -13,13 +13,13 @@ tags:
 
 # Instance methods
 
-You can programmatically change the internal state of Handsontable beyond what's possibile with props. To do that, you will need to call API methods of the relevant Handsontable instance associated with your instance of the `HotTable` component.
+You can programmatically change the internal state of Handsontable beyond what's possible with props. To do that, you will need to call API methods of the relevant Handsontable instance associated with your instance of the `HotTable` component.
 
 The following example implements the `HotTable` component showing how to reference the Handsontable instance from the wrapper component. 
 
 ::: example #example1 :react
 ```jsx
-import React, { useRef } from 'react';
+import { useRef } from 'react';
```

Comment:
It is misleading that this page uses `hotInstance.loadData` as an example of using a method, because the `data` prop, as presented on the [Binding to data](https://dev.handsontable.com/docs/next/react-data-grid/binding-to-data/#understand-binding-as-a-reference) page, is the preferable way of loading the data in React. 

This page should show more realistic use case of using instance methods in React, for example `hotInstance.selectCell(1, 1)`, which is already used in some other pages as an example of an API method call.

Reply:
changed

Reply:
I can't see the change. Did you forget to push?

Reply:
updated